web/src/pages/Environments/EnvsContent.tsx

```diff
@@ -0,0 +1,70 @@
+import {Form, Modal} from 'antd';
+import {Dispatch, SetStateAction, useCallback, useState} from 'react';
+import styled from 'styled-components';
+import SearchInput from '../../components/SearchInput';
+import {IEnvironment} from '../../redux/apis/TraceTest.api';
+import {EnvironmentForm} from './EnvironmentForm';
+import {EnvironmentState} from './EnvironmentState';
+import EnvList from './EnvList';
+import * as S from './Envs.styled';
+import EnvsActions from './EnvsActions';
+
+const CustomModal = styled(Modal)`
+  && {
+    .ant-modal-footer {
+      display: none;
+    }
+  }
+`;
+
+const EnvsModal = ({
```

Comment:
Let's move this to a separate file into the components folder

Reply:
ok ✅